src/tools/rbd/action/Bench.cc

```diff
@@ -456,7 +491,9 @@ void get_arguments_for_bench(po::options_description *positional,
   add_bench_common_options(positional, options);
 
   options->add_options()
-    ("io-type", po::value<IOType>()->required(), "IO type (read, write, or readwrite(rw))");
+    ("io-type", po::value<IOType>()->required(), "IO type (read, write, or readwrite(rw))")
+    ("pattern-byte", po::value<PatternByte>(),
+      "Which byte value to write (integer between 0-255, rand or rand-str [default: rand]");
```

Comment:
I think the convention is that option descriptions start with a lower case letter, except for abbreviations like IO.  Skimming through `src/test/cli/rbd/help.t`, I don't see any exceptions:
```suggestion
      "which byte value to write (integer between 0-255, rand or rand-str [default: rand]");
```